content/install-hook-openusercss.js

```diff
@@ -0,0 +1,178 @@
+'use strict';
+
+(() => {
+  const manifest = chrome.runtime.getManifest();
+  const allowedOrigins = [
+    'https://openusercss.org',
+    'https://openusercss.com'
+  ];
+
+  const sendPostMessage = message => {
+    allowedOrigins.forEach(origin => {
```

Comment:
`forEach` is not needed anymore.

Reply:
The reason I left it in is that `window.postMessage(message, origin);` needs an origin. If the forEach was removed, `origin` would become undefined.
What do you think would be the ideal solution to this?

Reply:
Simply use location.origin. Current code doesn't make sense.

Reply:
Oh, I see, thanks! This is what happens when I keep overthinking things. 🚀